services/venice-controller/src/main/java/com/linkedin/venice/controller/kafka/consumer/AdminConsumptionTask.java

```diff
@@ -356,8 +357,7 @@ public void run() {
                 logMessageFormat,
                 pubSubAdminTopic,
                 "Since this is the parent controller, it will be created now.");
-            admin.getTopicManager()
-                .createTopic(pubSubAdminTopic, 1, adminTopicReplicationFactor, true, false, minInSyncReplicas);
+            topicManager.createTopic(pubSubAdminTopic, 1, adminTopicReplicationFactor, true, false, minInSyncReplicas);
```

Comment:
I might be wrong, but the it seems like if the remote consumption is enabled, you will create topic in the remote region instead of local?

Reply:
you are right